static/css/components/work.less

```diff
@@ -42,7 +42,7 @@ div.editionAbout {
   }
 }
 
-.meta .object {
+.meta {
```

Comment:
not dd?

Reply:
Happy to revisit, if it's not a blocker, would love to get this changeset upstream